addons/udes_stock/tests/test_picking.py

```diff
@@ -376,3 +375,30 @@ def test08_get_suggested_location_by_product_all_lots(self):
         # and the location with matching product but different lot.
         self.assertEqual(len(suggested_locations), 2)
         self.assertEqual(self.test_location_01 + self.test_location_02, suggested_locations)
+
+    def test09_considers_partially_available_move_lines_when_suggesting_location(self):
```

Comment:
I realise the method name pretty much spells it out but please could we add a docstring briefly describing the tests as we do with the others?

Reply:
Added, please review